### DISCUSSION TITLE
Apply to Mozilla Recommended App program for Firefox

### DISCUSSION THREAD

**Discussion by sunjam** [floccusaddon/floccus#1678] [General]
Recommended extensions differ from [other extensions that are regularly reviewed by Firefox staff](https://support.mozilla.org/en-US/kb/add-on-badges) in that they are curated extensions that meet the highest standards of security, functionality, and user experience. Firefox staff thoroughly evaluate each extension before it receives Recommended status.

[Can I suggest extensions to become Recommended?](https://support.mozilla.org/en-US/kb/recommended-extensions-program)

Yes. If there’s extension you feel should be Recommended, please email amo-featured [at] mozilla [dot] org with a link to its AMO listing page.

**Comment by marcelklehr** [comment]
@sunjam Feel free to recommend floccus :blue_heart:
